spec/views/idv/in_person/ready_to_verify/show.html.erb_spec.rb

```diff
@@ -180,71 +202,13 @@
       end
     end
 
-    it 'renders Questions? and Learn more link only once' do
-      render
-
-      expect(rendered).to have_content(t('in_person_proofing.body.barcode.questions')).once
-      expect(rendered).to have_link(
-        t('in_person_proofing.body.barcode.learn_more'),
-        href: help_center_redirect_url(
-          category: 'verify-your-identity',
-          article: 'verify-your-identity-in-person',
-        ),
-      ).once
-    end
-
-    it 'renders the email sent success alert' do
-      render
-
-      expect(rendered).to have_content(t('in_person_proofing.body.barcode.email_sent'))
-    end
-
-    it 'template does not display Enhanced In-Person Proofing specific content' do
-      render
-
-      aggregate_failures do
-        [
-          t('in_person_proofing.headings.barcode_eipp'),
-          t('in_person_proofing.body.barcode.eipp_tag'),
-          t('in_person_proofing.headings.barcode_what_to_bring'),
-          t('in_person_proofing.body.barcode.eipp_what_to_bring'),
-          t('in_person_proofing.process.eipp_bring_id.heading'),
-          t('in_person_proofing.process.eipp_bring_id_with_current_address.heading'),
-          t('in_person_proofing.process.eipp_bring_id.info'),
-          t('in_person_proofing.process.real_id_and_supporting_docs.heading'),
-          t('in_person_proofing.process.real_id_and_supporting_docs.info'),
-          t('in_person_proofing.process.eipp_bring_id_plus_documents.heading'),
-          t('in_person_proofing.process.eipp_bring_id_plus_documents.info'),
-          t('in_person_proofing.process.eipp_state_id_passport.heading'),
-          t('in_person_proofing.process.eipp_state_id_passport.info'),
-          t('in_person_proofing.process.eipp_state_id_military_id.heading'),
-          t('in_person_proofing.process.eipp_state_id_military_id.info'),
-          t('in_person_proofing.process.eipp_state_id_supporting_docs.heading'),
-          t('in_person_proofing.process.eipp_state_id_supporting_docs.info'),
-          t('in_person_proofing.process.state_id.heading_eipp'),
-          t('in_person_proofing.process.state_id.info_eipp'),
-        ].each do |copy|
-          Array(copy).each do |part|
-            expect(rendered).to_not have_content(part)
-          end
-        end
+    context 'when Enhanced IPP is enabled' do
+      let(:is_enhanced_ipp) { true }
```

Comment:
Non-blocking: Again, I wonder if [Andrew's comment on the user mailer spec](https://github.com/18F/identity-idp/pull/11520#discussion_r1846901355) also applies to this set-up?